python/phylanx/ast/transformation.py

```diff
@@ -241,13 +254,20 @@ def _Assign(self, a, allowreturn=False):
         symbol_info = full_node_name(a)
         args = [arg for arg in ast.iter_child_nodes(a)]
         s = ""
-        if args[0].id in self.defs:
-            s += "store" + symbol_info
+
+        if is_node(args[0], 'Subscript'):
+            #print('Assign1', args[0], args[1])
+            s += "set" + symbol_info + "("
+            s += self.recompile(args[0]) + ", 0, 0, 0, 0, 1, 1, " + self.recompile(args[1]) + ")"
```

Comment:
What is the rationale to switch from using store() to using set()?

Reply:
@rtohid suggested testing it out. Is there a preference? I might try to start putting together some small documentation about PhySL primitives.

Reply:
I think this overlaps with what I did in fixing_440.

Reply:
@hkaiser store fails on execution when a slice is the target of the storage.

Reply:
@ct-clmsn If you wait for fixing_440 to be merged to master, then merge master with this branch, you can set array elements the way you want.

Reply:
@stevenrbrandt will hold out for fixing_440. thanks!